gcp/README.md

```diff
@@ -77,6 +77,16 @@ Users who already had an RtF email address/Google account usually have performed
 1. `rake plain_sh` is like `rake sh`, but not all configuration is performed. This can be helpful for debugging (e.g. when `rake sh` does not work) and with interactive commands.
 1. To `curl` a single couchdb instance: `kubectl exec --namespace gpii couchdb-couchdb-0 -c couchdb -- curl -s http://$TF_VAR_secret_couchdb_admin_username:$TF_VAR_secret_couchdb_admin_password@127.0.0.1:5984/`
 
+#### Accessing CouchDB and CouchDB Web UI 
+
+1. Running `rake couchdb_ui` will set up port-forwarding of CouchDB port to local
+   machine and print a link and credentials to access CouchDB Fauxton Web UI.
```

Comment:
Maybe add a point mentioning that credentials are ephemeral and will be regenerated on next `rake couchdb_ui` run?

Reply:
This will not really be true with the proposed changes, see updated PR description for detauls.